docs/graphcache/cache-updates.md

```diff
@@ -475,7 +475,7 @@ that imitates the result that the API is assumed to send back:
 ```js
 const cache = cacheExchange({
   optimistic: {
-    favoriteTodo(args, cache, info) {
+    favoriteTodo(variables, cache, info) {
       return {
         __typename: 'Todo',
         id: variables.id,
```

Comment:
Hiya :wave: While this should indeed read as `args`, this line would also need to be updated 😅❤️

Reply:
Hi Phil. Ah yes with the documentation above `args` is better. 
I chose `variables` like the template just below. Do you want me to change this one too?

I mean change this:
``` 
const cache = cacheExchange({
  optimistic: {
    favoriteTodo(variables, cache, info) {
      return {
        __typename: 'Todo',
        id: variables.id,
        favorite(args, cache, info) {
          return true;
        },
      },
    },
  },
});
```

to:

``` 
const cache = cacheExchange({
  optimistic: {
    favoriteTodo(args, cache, info) {
      return {
        __typename: 'Todo',
        id: args.id,
        favorite(args2, cache, info) { // Here I changed to `args2` but can let `args` as you want :)
          return true;
        },
      },
    },
  },
});
```

Reply:
I'd just call it `_args` and accept that shadowing the name even is fine ✌️ 
Ultimately, it's just a code sampel that conveys meaning, and `args` functionally over `variables` is more important than the exact name 🙌

Reply:
I made the changes ;)
Hope it's fine :D